src/assets/styles/themes/adamant/_buttons.scss

```diff
@@ -9,6 +9,10 @@
   text-transform: unset;
 }
 
+.round-btn {
```

Comment:
Remove this class. Icon buttons are rounded by default. Check Vuetify docs.

Reply:
I don't see how it's true. I tried using v-btn with v-icon in one tag but it's not rounded as in examples in docs. It's square with some border-radius.

Reply:
some app styles may affected the Vuetify styles